packages/server/src/organization/organization-course.entity.ts

```diff
@@ -17,22 +19,19 @@ export class OrganizationCourseModel extends BaseEntity {
 
   @Exclude()
   @ManyToOne(
-    type => OrganizationModel,
-    organization => organization.organizationCourses,
+    (type) => OrganizationModel,
+    (organization) => organization.organizationCourses,
   )
   organization: OrganizationModel;
 
   @Column({ nullable: true })
   organizationId: number;
```

Comment:
We can remove this since we already have a relationship with organizationmodel above

Reply:
This contradicts https://github.com/ubco-db/office-hours/pull/37#discussion_r1387331068